image/preseed/preseed_linux.go

```diff
@@ -291,7 +291,7 @@ func prepareCore20Mountpoints(prepareImageDir, tmpPreseedChrootDir, snapdSnapBlo
 	for _, dir := range []string{
 		"etc/udev/rules.d", "etc/systemd/system", "etc/dbus-1/session.d",
 		"var/lib/snapd/seed", "var/cache/snapd", "var/cache/apparmor",
-		"var/snap", "snap"} {
+		"var/snap", "snap", "var/lib/extrausers"} {
```

Comment:
nitpick about formatting:
```suggestion
		"var/snap", "snap", "var/lib/extrausers",
	} {
```